config/app.php

```diff
@@ -170,6 +170,7 @@
         /*
          * Application Service Providers...
          */
+        Bugsnag\BugsnagLaravel\BugsnagServiceProvider::class,
```

Comment:
Would you mind moving this up to the bottom of the "Package Service Providers" section?